pkgs/tools/networking/ipgrep/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, pythonPackages, makeWrapper }:
+
+pythonPackages.buildPythonApplication rec {
+  version = "1.0";
+  pname = "ipgrep";
+
+  src = fetchFromGitHub {
+    owner = "jedisct1";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "1qaxvbqdalvz05aplhhrg7s4h7yx4clbfd50k46bgavhgcqqv8n3";
+  };
+
+  phases = "unpackPhase installPhase fixupPhase";
```

Comment:
Don't assign phases directly, remove that and see what it takes to make it work.

Reply:
I patched the setup.py file, now it works without defining the phases.